docs/DeveloperGuide.md

```diff
@@ -229,6 +229,92 @@ user in the `s-update` command
     * Pros: It can can be extended more easily if there is a greater variety of drinks to store in the future.
     * Cons: There are not many operations to do with `Drink`s. It is only used to represent a constant set of
      drink types.
+     
+## \[Completed\] Finding sales data of some drinks
+  
+Finds specific drinks' sales data feature allows the user to get the sales data of a drink quickly. The command is:
+  
+* `s-find KEYWORDS [MORE_KEYWORDS]` - Views sales data of drinks with the specified keywords.
+  
+#### Completed Implementation
+  
+The completed finds sales data of some drinks mechanism is facilitated by `InputContainsKeywordsPredicate`. It implements 
+Predicate<SalesRecordEntry>. 
+
+It exposes to `#Model updateFilteredSalesList(Predicate<SalesRecordEntry> predicate)`.
+  
+Given below is an example usage scenario and how the find drinks' sales data mechanism behaves at each step.
+  
+Step 1. The user launches the application. If the storage file for the sales book is empty, `SalesBook` will 
+be initialized with the six pre-defined drinks, namely `BSBM`, `BSBBT`, `BSBGT`, `BSPM`, `BSPBT` and `BSPGT`
+with the sales data of 0 for all. If the storage file for the sales book is not empty, `SalesBook` will read the  
+data from the storage file.
+  
+Step 2. The user executes `s-find BSBBT` to view BSBBT's current sales data. The `s-find BSBBT` command is
+parsed by `SalesFindCommandParser` which parses the input to get the matched drink's name and 
+returns an  `SalesFindCommand`,  which returns the drinks their sales data.
+  
+The following activity diagram shows how the find drink's sales data operation works:
+![Find Drink's Activity Diagram](images/SalesFindActivityDiagram.png) 
+
+#### Design consideration:
+  
+##### Aspect: How to find drink's sales data
+  
+  * **Alternative 1 (current choice):** Obtain the drink's name entered by the user, and use the
```

Comment:
Since you only stated one implementation, calling it `Alternative 1` may not be that good? Since there was no other alternatives considered.

Reply:
hh ok I copied the format from original UG. I will delete alternative one here.